packages/api/src/api/users.ts

```diff
@@ -0,0 +1,40 @@
+import { Router } from 'express';
+import { User } from '../entities/User';
+import logger from '../logger';
+
+export const users = Router();
+
+// const publicFields: (keyof typeof User)[] = ['pronouns', 'wat'];
+const stripSensitiveFields = (user: User): Partial<User> => ({
+  name: user.name,
+  pronouns: user.pronouns,
+  schoolName: user.schoolName,
+});
+
+users.get('/:userId', async (req, res) => {
+  const { userId } = req.params;
+
+  try {
+    // Check if userId is in correct format
+    if (Number.isNaN(userId)) {
```

Comment:
You would have caught this in tests, but `Number.isNaN` only checks if it's exactly `NaN` and since `req.params` are only strings, it'll always be false. You need to convert it to a number first so that it'll turn into `NaN` if it's wrong.

![image](https://user-images.githubusercontent.com/7807353/136818967-51e74483-6fd2-499c-b007-b8691f30ee65.png)


```suggestion
    if (Number.isNaN(Number(userId))) {
```